pkg/api/handlers/compat/ping.go

```diff
@@ -18,7 +18,7 @@ import (
 func Ping(w http.ResponseWriter, r *http.Request) {
 	// Note: API-Version and Libpod-API-Version are set in handler_api.go
 	w.Header().Set("BuildKit-Version", "")
-	w.Header().Set("Builder-Version", "")
+	w.Header().Set("Builder-Version", "1")
```

Comment:
Can you add a comment here what the value 1 means. Looking in the docker API doc I was unable to quickly find a description. I know it was described in your document but it would be good to get some of that context into the comments/commit messages.